docs/development/how-to-add-a-type-with-bindings.md

```diff
@@ -0,0 +1,10 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+# How to add a type with its python bindings
+
+1. Define it in the `libparsec/crates/types`
+2. Implement (or derive) Debug, Copy, Clone, Hash, Serialize, Deserialize, PartialEq, Eq, FromStr, ToString
```

Comment:
We do you get that list ? seems strange to require implementing `Hash`, `Copy`, `FromStr` for all types

Reply:
@vxgmichel I did receive any reply, why did you close that ?

Reply:
Ooops I thought it was solved by @AureliaDolo since the snippet as marked as `Outdated`

Reply:
Yes, I got carried away by the outdated

Reply:
I did as it was for InvitationType, should have checked with other impl